packages/core/src/lib/TwaManifest.ts

```diff
@@ -120,6 +129,12 @@ export class TwaManifest {
     this.display = asDisplayMode(data.display!) || DEFAULT_DISPLAY_MODE;
     this.themeColor = new Color(data.themeColor);
     this.navigationColor = new Color(data.navigationColor);
+    this.navigationColorDark = new Color(data.navigationColorDark == undefined ?
+      DEFAULT_NAVIGATION_COLOR : data.navigationColorDark);
+    this.navigationDividerColor = new Color(data.navigationDividerColor == undefined ?
+      DEFAULT_NAVIGATION_DIVIDER_COLOR : data.navigationDividerColor);
+    this.navigationDividerColorDark = new Color(data.navigationDividerColorDark == undefined ?
+      DEFAULT_NAVIGATION_COLOR : data.navigationDividerColorDark);
```

Comment:
This can be `new Color(data.navigationDividerColorDark ?? DEFAULT_NAVIGATION_COLOR)`.